src/tests/suite/io_conf/daos_generate_io_conf.c

```diff
@@ -671,6 +671,7 @@ main(int argc, char **argv)
 	if (obj_class && obj_class != default_class)
 		D_FREE(obj_class);
 
-	close(fd);
+	if (fd > 0)
```

Comment:
technically, 0 is a valid descriptor so this should be >= 0.  It's normally stdout so unlikely but is a valid descriptor.